src/components/LenderProfile/LenderDedicationsList.vue

```diff
@@ -135,12 +153,14 @@ export default {
 			this.smoothScrollTo({ yPosition: topOfSectionToScrollTo, millisecondsToAnimate: 750 });
 		}
 	},
-	mounted() {
-		this.fetchLenderDedications();
-	},
 	created() {
 		this.pageQuery = _get(this.$route, 'query');
 		this.updateFromParams(this.pageQuery);
-	}
+	},
+	watch: {
+		isLoading() {
+			if (!this.isLoading) this.fetchLenderDedications();
```

Comment:
It looks like the lazy loading pattern here isn't quite lazy loading but instead other components all load once the `LenderLoansList` is done loading. As mentioned in the ticket, you can use the `@visible="fetchAsyncData"` such as in `YourTeams.vue` to do lazy loading.